libs/postgres_ffi/src/xlog_utils.rs

```diff
@@ -329,8 +329,8 @@ impl CheckPoint {
     ///
     /// Returns 'true' if the XID was updated.
     pub fn update_next_xid(&mut self, xid: u32) -> bool {
-        // nextXid should nw greater than any XID in WAL, so increment provided XID and check for wraparround.
-        let mut new_xid = std::cmp::max(xid + 1, pg_constants::FIRST_NORMAL_TRANSACTION_ID);
+        // nextXid should be greater than any XID in WAL, so increment provided XID and check for wraparround.
+        let mut new_xid = std::cmp::max(xid.wrapping_add(1), pg_constants::FIRST_NORMAL_TRANSACTION_ID);
```

Comment:
Does this actually change the behavior? My understanding was that Rust's types already do wrapping by default. Maybe we want checked_add?

Reply:
Rust is wonderful language. Integer overflow cause panic in debug mode and just silent wraparound in release mode.

No, we don't want checked_add. Overflow is valid and should not be treated as error.

Reply:
Got it, thanks